samples/deep_hierarchy.html

```diff
@@ -22,22 +22,28 @@
           'A',
           {
             label: 'B1',
+            expand:true,
```

Comment:
can you create a separate example instead of adapting existing ones?

Reply:
done in reverse_order.html